### PR TITLE
feat: refund step for reverse swaps

### LINKED ISSUE #89: [UX Revamp] Refund file download improvements
User feedback:
> a noob user could easily miss this file...

### PR DESCRIPTION
**Pull request by dni**
closes #89

todo:
- [x] retranslate german and esp

### CODE REVIEW

**Comment by kilrau** [comment]
As just discussed:
- remove all other download refund file buttons, apart from failure pages like "lockup failed" or "invoice failed to pay"
- redesign this step to:
Text 
left: skip download, maybe even without color
right: download

Download should also automatically go to next step.

Title: Download Refund File
If this swap fails, you'll have to refund your chain bitcoin. Refund information is additionally stored in your browser's local storage but can be lost if
* you are using incognito mode
* you are using Tor Browser
* clear your browsing history
⚠️
If any of the above applies to you, we strongly recommend downloading this refund file!
Skip Download | Download Refund File

**Review by michael1011 — CHANGES_REQUESTED** [review]
I already said a couple times that I don't think this is a good idea and we need to come up with something smarter than being obnoxious about downloading countless files. The more warnings we show everywhere, the less people are gonna care. Every obnoxious warning you show and extra click you introduce is a fail in UX.

One thing that could make this PR bearable is checking the history in the local storage and not showing it if the user has done swaps in the past with that browser.


But since you guys insist on this: concept ack

**Comment by dni** [comment]
if a user in a browser without persistent storage makes 2 swaps in one session, we only show him the download refund once. 
i would keep it everytime, its so simple to click 1 button more and saves you from loosing funds. worth it.

File: src/i18n/i18n.js [hunk 1]
```diff
@@ -213,6 +221,14 @@ const dict = {
         copy_amount: "Betrag kopieren",
         copy_bip21: "BIP21 kopieren",
         copied: "In die Zwischenablage kopiert!",
+        backup_skip: "Überspringen",
+        backup_refund: "Rückerstattungsdatei herunterladen",
+        backup_refund_subline:
+            "Lade deine Rückerstattungsdatei herunter und bewahre sie sicher auf. Du benötigst diese Datei, um deine Bitcoin zurückzuerhalten, falls der Swap fehlschlägt.",
+        backup_refund_list:
+            "are using incognito mode\nare using tor browser\nclear your browsing history",
```
Comment (michael1011) [inline src/i18n/i18n.js:229]: Come one. Please check the translations before committing

File: src/i18n/i18n.js [hunk 2]
```diff
@@ -356,6 +372,14 @@ const dict = {
...
         copy_bip21: "Copiar BIP21",
         copied: "Copiado",
+        backup_skip: "Omitir",
+        backup_refund: "Descargar archivo de reembolso",
+        backup_refund_subline:
+            "Descargue el archivo de reembolso para recuperar sus fondos en caso de que algo salga mal",
+        backup_refund_list:
+            "están usando el modo incógnito\nestán usando el navegador Tor Borra\ntu historial de navegación",
+        backup_refund_skip:
+            "Si no desea descargar el archivo de reembolso, puede omitir este paso",
```
Comment (michael1011) [inline src/i18n/i18n.js:382]: @kilrau pls check the spanish translations

**Review by michael1011 — CHANGES_REQUESTED** [review]

File: src/i18n/i18n.js (hunk 1 above)
Comment (dni) [reply src/i18n/i18n.js:229]: ???

File: src/i18n/i18n.js [hunk 3]
```diff
@@ -75,6 +75,14 @@ const dict = {
         copy_amount: "Copy amount",
         copy_bip21: "Copy BIP21",
         copied: "Copied to clipboard!",
+        backup_skip: "Skip download",
+        backup_refund: "Download Refund File",
+        backup_refund_subline:
+            "If this swap fails, you'll have to refund your chain bitcoin. Refund information is additionally stored in your browser's local storage but can be lost if you",
```
Comment (kilrau) [inline src/i18n/i18n.js:81]:
```suggestion
            "If this swap fails, you'll have to refund your chain bitcoin using your refund key. This key is stored in your browser's local storage by default, but can be lost if you",
```

File: src/i18n/i18n.js [hunk 4]
```diff
@@ -75,6 +75,14 @@ const dict = {
...
+        backup_refund_list:
+            "are using incognito mode\nare using tor browser\nclear your browsing history",
```
Comment (kilrau) [inline src/i18n/i18n.js:83]: no `/n` in the string please, split them up into 3

File: src/i18n/i18n.js [hunk 5]
```diff
@@ -213,6 +221,14 @@ const dict = {
         copy_amount: "Betrag kopieren",
         copy_bip21: "BIP21 kopieren",
         copied: "In die Zwischenablage kopiert!",
+        backup_skip: "Überspringen",
+        backup_refund: "Rückerstattungsdatei herunterladen",
+        backup_refund_subline:
+            "Lade deine Rückerstattungsdatei herunter und bewahre sie sicher auf. Du benötigst diese Datei, um deine Bitcoin zurückzuerhalten, falls der Swap fehlschlägt.",
```
Comment (kilrau) [inline src/i18n/i18n.js:227]:
```suggestion
            "Falls dieser Swap fehlschlagen sollte, musst du dir deine Bitcoin mit deinem Rückerstattungsschlüssel zurückholen. Dieser Schlüssel wird standardmäßig lokal in dein Browser gespeichert, kann aber verloren gehen wenn",
            
```

File: src/i18n/i18n.js (hunk 1 above)
Comment (kilrau) [inline src/i18n/i18n.js:229]: English please and split up as per above

File: src/i18n/i18n.js [hunk 6]
```diff
@@ -213,6 +221,14 @@ const dict = {
...
+        backup_refund_list:
+            "are using incognito mode\nare using tor browser\nclear your browsing history",
+        backup_refund_skip:
+            "Du kannst diese Datei später herunterladen, indem du auf den Button in der oberen rechten Ecke klickst.",
```
Comment (kilrau) [inline src/i18n/i18n.js:231]: If any of the above applies to you, we strongly recommend downloading this refund file!
```suggestion
            "Falls einer dieser Punkte zutrifft, empfehlen wir ausdrücklich die Rückerstattungsdatei herunterzuladen!",
```

File: src/i18n/i18n.js [hunk 7]
```diff
@@ -356,6 +372,14 @@ const dict = {
         copy_amount: "Copiar monto",
         copy_bip21: "Copiar BIP21",
         copied: "Copiado",
+        backup_skip: "Omitir",
```
Comment (kilrau) [inline src/i18n/i18n.js:375]:
```suggestion
        backup_skip: "Saltar",
```

File: src/i18n/i18n.js [hunk 8]
```diff
@@ -356,6 +372,14 @@ const dict = {
...
+        backup_refund: "Descargar archivo de reembolso",
+        backup_refund_subline:
+            "Descargue el archivo de reembolso para recuperar sus fondos en caso de que algo salga mal",
```
Comment (kilrau) [inline src/i18n/i18n.js:378]:
```suggestion
            "Si este intercambio falla, tendrás que reembolsar tu bitcoin utilizando tu clave de reembolso. Esta clave se guarda por defecto en el almacenamiento local de tu navegador, pero puede perderse si",
```

File: src/i18n/i18n.js [hunk 9]
```diff
@@ -356,6 +372,14 @@ const dict = {
...
+        backup_refund_list:
+            "están usando el modo incógnito\nestán usando el navegador Tor Borra\ntu historial de navegación",
```
Comment (kilrau) [inline src/i18n/i18n.js:380]:
```suggestion
            "utiliza el modo incógnito\nutiliza el navegador tor\nborra tu historial de navegación",
```

File: src/i18n/i18n.js (hunk 2 above)
Comment (kilrau) [inline src/i18n/i18n.js:382]:
```suggestion
            "Si alguno de los puntos anteriores le afecta, le recomendamos que descargue el archivo de reembolso.",
```

**Review by kilrau — CHANGES_REQUESTED** [review]
Added most of the string issues als suggestions, but still need to break up the bullet point list into 3 independent strings.

**Pull request merged**